Danielshow/Fast-Food
# Restrict admin from seeing profile page and placing orders #161300484

## What does this PR do?
This PR restrict admin from visiting order-list page and profile page. These pages are only for users

## Description of Task to be completed
- Get if the token provided is an admin
- Redirect them when they go to the pages

## How should this be manually tested
```
- Go to the website
- Register using valid admin credentials
- click on /order-list.html
-  it should redirect to admin-page.html
```
#### What are the relevant pivotal stories 
## Narrative:
An admin, Whenever I click on profile page, it should lead me to admin-page
 
## Acceptance Criteria:

Scenario: 
Given an admin
  And Login with Valid credentials
When I click to profile link
Then  I should be redirected to admin-page
## Branch name  
ft-profile-page-clicked-if-user-#161300484
